geomesa-jupyter/src/main/scala/org/locationtech/geomesa/jupyter/Leaflet.scala

```diff
@@ -31,6 +40,52 @@ object L {
        """.stripMargin
   }
 
+  /**
+    * Takes a string containing a valid javascript styling function as its argument
+    * @param javascriptFunction The javascript styling function
+    */
+  case class StyleOptionFunction(javascriptFunction: String) extends StyleOption {
+    def render: String = javascriptFunction
+  }
+
+
+  private object OnFeatureClick {
+    def render: String =
+      """
+       |function onClick(e) {
+       |  e.target.openPopup();
+       |}
+       |
+       |function onFeature(feature, layer) {
+       |  var keys = Object.keys(layer.feature.properties);
+       |  var str = "";
+       |  for (var i in keys) {
+       |    var key = keys[i];
+       |    if (key != "geom") {
```

Comment:
I fixed this issue removing that bit of logic and by adding ```!b.isInstanceOf[Geometry]``` to the call on line 145:
```
sf.getType.getAttributeDescriptors.zip(sf.getAttributes)
   .filter{case(a,b) => b != null && !b.isInstanceOf[Geometry]}
   .map { case (d, a) => propToJson(d, a) }.mkString(sep =",\n")
```